mods/ra/rules/structures.yaml

```diff
@@ -338,6 +340,21 @@ PDOX:
 		Duration: 20
 		KillCargo: yes
 		DisplayRadarPing: True
+	ChronoshiftPower@advancedchronoshift:
+		OrderName: AdvancedChronoshift
+		Prerequisites: structures.germany
```

Comment:
Good point.  We can fix both sets of problems by having the chronosphere grant its own race-specific prerequisite, in the same way that we grant structures.faction, etc.